__init__.py

```diff
@@ -56,6 +55,13 @@ def __init__(self):
         self.volume_sound = join(dirname(__file__), "blop-mark-diangelo.wav")
         self.vol_before_mute = None
         self._mixer = None
+        volume_words = self.translate_namedvalues('volume.words', delim=',')
+        if volume_words != {}:
```

Comment:
empty dicts are Falsey, and populated dicts are Truthy - so this can probably be simplified to:
`if volume_words:`